### PR TITLE
Will-R-JS-Roman-Numerals

### PR DESCRIPTION
**Pull request by wratterman**

### CODE REVIEW

File: test/new-roman-test.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+const assert = require('chai').assert
+const convertToNewRoman = require('../lib/new-roman.js')
+
+describe('convertToNewRoman functionality', function() {
+  context('convertToNewRoman function', function(){
+    it('it can convert a number to new Roman Numerals', function(){
```
Comment (case-eee) [inline test/new-roman-test.js:6]: i suggest breaking each of these use cases into separate `it` blocks to make this more organized and readable. i love that you're testing your edge cases (0 and 3001) 👍

File: lib/new-roman.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+const arabicToRoman = require('../lib/shared.js').arabicToRoman
+
+const newRomanNumerals = {
```
Comment (case-eee) [inline lib/new-roman.js:3]: love this!

File: lib/shared.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+function arabicToRoman(romanNumerals, number) {
```
Comment (case-eee) [inline lib/shared.js:1]: love that this works with different sets of conversions

**Review by case-eee — COMMENTED** [review]
@wratterman this is AWESOME. great job 🎉
